### PR TITLE
fix: paddings on settings pages

### PR DESCRIPTION
**Pull request by martinkaintas**

### CODE REVIEW

File: src/popup/pages/ErrorLogSettings.vue [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default defineComponent({
 @use '../../styles/typography';
 
 .errors-log-settings {
-  padding: var(--screen-padding-x);
+  padding: 0 var(--screen-padding-x);
```
Comment (peronczyk) [inline src/popup/pages/ErrorLogSettings.vue:50]:
```suggestion
  padding-block: var(--screen-padding-x);
```

File: src/popup/pages/LanguageSettings.vue [hunk 2]
```diff
@@ -62,7 +62,7 @@ export default defineComponent({
 @use '../../styles/typography';
 
 .language-settings {
-  padding: var(--screen-padding-x);
+  padding: 0 var(--screen-padding-x);
```
Comment (peronczyk) [inline src/popup/pages/LanguageSettings.vue:65]:
```suggestion
  padding-block: var(--screen-padding-x);
```

File: src/popup/pages/Networks.vue [hunk 3]
```diff
@@ -77,7 +77,7 @@ export default defineComponent({
 @use '@/styles/typography';
 
 .networks {
-  padding: var(--screen-padding-x);
+  padding: 0 var(--screen-padding-x);
```
Comment (peronczyk) [inline src/popup/pages/Networks.vue:80]:
```suggestion
  padding-block: var(--screen-padding-x);
```

**Comment by peronczyk** [comment]
Instead of:
```
  padding: 0 var(--screen-padding-x);
```
I suggest to use:

```suggestion
  padding-block: var(--screen-padding-x);
```

**Review by peronczyk — APPROVED** [review]

**Comment by martinkaintas** [comment]
> Instead of:
> 
> ```
>   padding: 0 var(--screen-padding-x);
> ```
> 
> I suggest to use:
> 
> ```
>   padding-block: var(--screen-padding-x);
> ```

This doesn't work, but `padding-inline` works

**Pull request merged**